.github/workflows/build-test.yml

```diff
@@ -176,6 +176,9 @@ jobs:
 
     - name: Build Windows with maturin on Python ${{ matrix.python }}
       if: matrix.os.matrix == 'windows'
+      env:
+        CC: 'clang'
+        CFLAGS: "-D__BLST_PORTABLE__"
```

Comment:
I would expect this to be defined based on the crate feature already, but can't hurt